Metadata/CIP-Certification-Metadata.md

```diff
@@ -0,0 +1,466 @@
+          },
+          "fullScriptHash": "384da5375661cb1e07713eea236cd681921dcb4e80d7cd0ed4e7feebd7",
+          "scriptHash": "6cd68191dcb4e80d7c5661cb1e074e7feebd73eea232d0ed384da537",
+          "contractAddress": "addr1wywukn5q6lrdntgrysv0f7a0tna63ymffh23ms5uxsaesuk8s8fel"
+        }
+  ]
+}
+```
+
+[TO DISCUSS: Verification tools, how do we represent that? with versions, compilation options]
+
+The metadata should be discoverable by all certification stakeholders, including end-users, DApp developers, and ecosystem components, such as light wallets and DApp stores. Information should be indexable by certification issuer, DApp developer, DApp and DApp version.
+
+It should be possible for there to be multiple versions of metadata published by the same certification issuer for the same version of a DApp, particularly when a (minor) update of the evidence is necessary. In such a case it should be possible to identify the most recent version of the report for that DApp version. It will also be the case that the same DApp may have certification metadata provided by multiple different certification issuers.
+
+It should be possible for wallets to identify to users the certification status of a DApp when they are signing a transaction that is being submitted to a deployed DApp.
+
+### Custom fields
+Certification issuers should be free to add additional fields to fit some additional needs.
+
+
+### Certification issuers
+| Certification issuer | URL                | Contact email        | Certification levels | Public key                                |
+|----------------------|-----               |---------------       |----------------------|------------                               |
+| Example Ltd.         | http://example.com | contact@example.com  | 1,2                  | EXAMPLEPUBLICKEYPGPED25519 (PGP, ed25519) |
+ 
+## Rationale: how does this CIP achieve its goals?
+
+An on-chain solution is preferred as it allows for it to be checkable by any stakeholder and immutable.
+
+Certificate are issued by certification issuers that sign the certificate to prevent certificate forgeries.
+This design allows for anyone to issue certificates as long as they sign it but stakeholders are then free to maintain a list of the trusted certification issuers.
+
+These certificates are self-standing and can be presented as-is by any stakeholder.
+
+This proposal does not affect any backward compatibility of existing solution but is based on the work done for CIP72 on DApp registration and Discovery. It is also linked to CIP52 on Cardano audit best practices guidelines.
+
+### Other designs considered
+**Updates to registration entries**
+This would have required DApp owner or certification issuers to add certificates to every registration making it harder to maintain a shared state between all stakeholders. The chosen design requires to follow the chain to discover the certificates which should be expected from stakeholders.
```

Comment:
These two lines will appear after one another in HTML, and therefore we should have a separator. A colon maybe?